app/src/main/java/com/github/sdp_begreen/begreen/fragments/CameraContainer.kt

```diff
@@ -0,0 +1,28 @@
+package com.github.sdp_begreen.begreen.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.add
+import androidx.fragment.app.commit
+import com.github.sdp_begreen.begreen.R
+
+class CameraContainer : Fragment() {
+    companion object {
+        fun newInstance() = CameraContainer()
```

Comment:
This linke is not tested, maybe you could try to instantiate this fragment through the `newInstance` method to increase the coverage, or simply remove the `companion object` as it is not very useful.